acceptance/exec/exec_test.go

```diff
@@ -0,0 +1,33 @@
+package exec_test
```

Comment:
We can skip this file for now, there's a separate ticket for the exec acceptance tests

Reply:
Removed